doc/classes/AudioStreamPlayer3D.xml

```diff
@@ -265,67 +269,94 @@
 			<return type="void">
 			</return>
 			<description>
+				Stops the audio.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="area_mask" type="int" setter="set_area_mask" getter="get_area_mask">
+			Areas in wich this sound plays.
 		</member>
 		<member name="attenuation_filter_cutoff_hz" type="float" setter="set_attenuation_filter_cutoff_hz" getter="get_attenuation_filter_cutoff_hz">
+			Dampens audio above this frequency in hz.
 		</member>
 		<member name="attenuation_filter_db" type="float" setter="set_attenuation_filter_db" getter="get_attenuation_filter_db">
+			Amount in db how much the filter affects the loudness.
 		</member>
 		<member name="attenuation_model" type="int" setter="set_attenuation_model" getter="get_attenuation_model" enum="AudioStreamPlayer3D.AttenuationModel">
+			Decides if audio should get quieter with distance linearly, quadratically or logarythmically.
 		</member>
 		<member name="autoplay" type="bool" setter="set_autoplay" getter="is_autoplay_enabled">
+			If 'true' audio plays from start of the game.
```

Comment:
`If [code]true[/code], ...` with code tag and a comma between cause and effect. Also default value.